src/app/pagination/pagination.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="content-view-pf-pagination list-view-pf-pagination clearfix">
+  <div class="form-group">
+    <div class="padding-right-10">
+      <div class="btn-group dropdown" dropdown>
+        <button class="btn btn-default dropdown-toggle" dropdownToggle>
+        {{config.pageSize}}
+        <span class="caret"></span>  
+      </button>
+        <ul class="dropdown-menu" *dropdownMenu>
+          <li *ngFor="let increment of config.pageSizeIncrements" [ngClass]="{'selected': increment === config.pageSize}" class="display-length-increment">
+            <a role="menuitem" (click)="onPageSizeChange(increment)">{{increment}}</a></li>
+        </ul>
+      </div>
+    </div>
+    <span class="per-page-label">per page</span>
+  </div>
+  <div class="form-group">
+    <span><span class="pagination-pf-item-current">{{getStartIndex()}} - {{getEndIndex()}}</span> of <span class="pagination-pf-items-total">{{config.numTotalItems}}</span></span>
+    <ul class="pagination pagination-pf-back">
+      <li [ngClass]="{'disabled': config.pageNumber === 1}"><a title="First Page" (click)="gotoFirstPage()" class="goto-first-page"><span class="i fa fa-angle-double-left"></span></a></li>
+      <li [ngClass]="{'disabled': config.pageNumber === 1}"><a title="Previous Page" (click)="gotoPreviousPage()" class="goto-prev-page"><span class="i fa fa-angle-left"></span></a></li>
```

Comment:
For completeness, I'd like to see gotoFirstPage($event), gotoPreviousPage($event), gotoNextPage($event), gotoLastPage($event), and onPageSizeChange($event, increment) where MouseEvent is provided as the $event parameter. Even if the MouseEvent object is not ultimately used, it's clear that a mouse click invoked these functions.